monkey/monkey_island/cc/ui/src/components/ui-components/checkbox.js

```diff
@@ -0,0 +1,66 @@
+import '../../styles/Checkbox.scss'
+import React from 'react';
+
+class Checkbox extends React.PureComponent {
```

Comment:
I'll try, but the last time I renamed a file just by changing to capital letter I had a lot of trouble, because windows fs doesn't recognise the change, but git does, so I got two files commited, one with an uppercase letter and one with the lowercase letter instead of just one....

Reply:
you can do it in 2 parts. rename to a different name, then rename to the original name with a capital letter.
Alternatively do the change from linux or from github's UI